tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -43,16 +43,6 @@
     - include_tasks: restore_backup.yml
     when: he_restore_from_file is defined and he_restore_from_file
   rescue:
-    - name: Fetch logs from the engine VM
-      include_tasks: fetch_engine_logs.yml
-      ignore_errors: yes
-      delegate_to: "{{ he_ansible_host_name }}"
```

Comment:
Please notice that include_tasks and delegate_to could not be combined together with the desired results neither before as for https://github.com/ansible/ansible/issues/37995
while now it's directly failing on ansible 2.8 as for https://github.com/kubernetes-sigs/kubespray/issues/3815